pet-kata-solutions/src/test/java/org/eclipse/collections/petkata/Exercise5Test.java

```diff
@@ -70,6 +72,25 @@ public void addPetAgesToExistingSet()
         Assertions.assertEquals(expected, petAges);
     }
 
+    @Test
+    @Tag("SOLUTION")
+    @DisplayName("findOwnerWithMoreThanOnePetOfTheSameType - 🐹 🐹")
+    public void findOwnerWithMoreThanOnePetOfTheSameType()
+    {
+
+        Person petOwner= this.people.detect( p-> {
```

Comment:
We use different formatting elsewhere. The brackets should start on a new line. You might also need to adjust your formatting settings in idea.